templates/Dockerfile.template

```diff
@@ -21,7 +31,21 @@ RUN set -x \
 	&& wget -q -O /usr/local/src/gosu.tar.gz "https://github.com/tianon/gosu/archive/$GOSU_VERSION.tar.gz" \
 	# extract gosu binary and check signature
 	&& export GNUPGHOME="$(mktemp -d)" \
-	&& gpg --keyserver ha.pool.sks-keyservers.net --recv-keys {{ $gosu.Gpg }} \
+	&& found='' \
+	&& for server in \
```

Comment:
I don't think we're currently do this in other repos. Is this a pattern we're gonna established into the future?